android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -111,6 +118,146 @@ public void testEnableAndDisableCustomLogging() {
         assertEquals(4, customLogger.getLines().size());
     }
 
+    @Test @Ignore
```

Comment:
ignored since the debug logs are not printing anything in it

Reply:
So why ignore instead of adjusting the test?  For release mode it's true no debug logs should be written but in debug mode the debug logs should be written.

Reply:
waiting to discuss with Pasin tomorrow, and need to understand how the debug and release works in Android.

Reply:
* Android Studio has a separate option/button for run (release build type) and debug. To enable debug build type, select debug.
* I understand that when running test from Jenkins using gradle command, the release build type will be used.
* As @borrrden suggested, I think we should modify the test to ignore or not checking debug.

Reply:
I used below code to check whether the test is in debug or not. and I am getting `isDebug=true` but still, `cbl_debug_xxx` file only with heading. 

any suggestions

```
boolean isDebug = (
                        (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0
```

Reply:
I guessed CMAKE_BUILD_TYPE doesn't get set correctly according to the android build type.

We may need to explicitly set the CMAKE_BUILD_TYPE to the right value in https://github.com/couchbase/couchbase-lite-android/blob/master/android/CouchbaseLite/build.gradle#L86-L97.

Reply:
i will merge the PR, and create a separate issue to fix the test `testFileLoggingMaxSize`. So that we don't need to keep other tests.

Reply:
referencing issue #1835 and resolving this conversation.